frontend/src/components/MapView/LeftPanel/layersPanel/AnalysisLayerSwitchItem/index.tsx

```diff
@@ -136,17 +136,19 @@ const AnalysisLayerSwitchItem = memo(
       }
       return (
         <Tooltip title="Opacity">
-          <IconButton
-            disabled={!selected}
-            classes={{
-              root: isOpacitySelected
-                ? classes.opacityRootSelected
-                : classes.opacityRoot,
-            }}
-            onClick={handleOpacityClick}
-          >
-            <OpacityIcon />
-          </IconButton>
+          <>
```

Comment:
I add this fragment (in several files) to fix this frontend error message
```
Material-UI: You are providing a disabled `button` child to the Tooltip component.
A disabled element does not fire events.
Tooltip needs to listen to the child element's events to display the title.

Add a simple wrapper element, such as a `span`.
```

Reply:
are you sure they are all needed?